core/domain/subscription_services.py

```diff
@@ -104,6 +104,20 @@ def unsubscribe_from_creator(user_id, creator_id):
         subscriptions_model_user.put()
 
 
+def get_all_threads_subscribed_to(user_id):
+    """Returns a list with ids of all the threads to which the user is
+    subscribed.
+
+    Callers of this function should ensure that the user_id is valid.
+    """
+    subscriptions_model = user_models.UserSubscriptionsModel.get(
+        user_id, strict=False)
+
+    return (
+        subscriptions_model.feedback_thread_ids
```

Comment:
Ah ok, thanks. Maybe say "all the feedback and suggestion threads" in the docstring, then -- that at least helps guard a bit against the misnaming here (which is hard to change).